### PR TITLE
Search working again and clean up code

### PR DESCRIPTION
**Pull request by mjvalade**

### CODE REVIEW

File: app/components/Search.js [hunk 1]
```diff
@@ -1,75 +1,61 @@
...
+            onPress={this._onCallApi.bind(this)}
+          >
+            <Text style={styles.buttonText}>Search for Meteors</Text>
+          </TouchableHighlight>
+          {/* {spinner = this.state.isLoading ?
+            (<ActivityIndicator
+                style={styles.spinner}
+                size='large'
+                color='red'
+            /> ) : */}
```
Comment (kylem038) [inline app/components/Search.js:47]: What this commented out code?

File: app/components/Visuals.js [hunk 2]
```diff
@@ -28,23 +28,21 @@ class Visuals extends Component {
...
-          style={styles.visuals}
-        >
-          <Text style={styles.title}>Meteorite Mass</Text>
-          <MassChart meteors={this.props.meteors}/>
-        </View>
-        <View>
-          <Text style={styles.text}>Meteors Per Year</Text>
-          <FellPerYearChart style={styles.chart} meteors={this.props.meteors}/>
-        </View>
+        </TouchableHighlight> */}
```
Comment (kylem038) [inline app/components/Visuals.js]: Also here?

**Review by kylem038 — COMMENTED** [review]
Thanks for cleaning up all my whitespace issues!

File: app/components/Search.js (hunk 1 above)
Comment (mjvalade) [reply app/components/Search.js:47]: This is to have a spinner to show data loading when API call made. Needs more logic to know when data returns from NASA.

File: app/components/Visuals.js (hunk 2 above)
Comment (mjvalade) [reply app/components/Visuals.js]: Navigation button, removed.

**Pull request merged**